### DISCUSSION TITLE
Has anybody already used sling with greenplum as a target with the postgres driver ?

### DISCUSSION THREAD

**Discussion by fabrice-etanchaud** [slingdata-io/sling-cli#235] [Q&A]
Has anybody already used sling with greenplum as a target with the postgres driver ?

**Comment by flarco** [comment]
Don't know much about greenplum. Have you tried to use sling to access it (via postgres credential)?

**Reply by fabrice-etanchaud** [reply]
As soon as our greenplum cluster is up, I will give sling a try, and tell you.
Thank you !
